driver-core/src/test/java/com/datastax/driver/core/SchemaChangesTest.java

```diff
@@ -482,8 +484,7 @@ public void should_notify_of_keyspace_drop(String keyspace) throws InterruptedEx
         }
         assert added != null;
         refreshSchema();
-        for (Metadata m : metadatas())
-            assertThat(m.getReplicas(keyspace, Bytes.fromHexString("0xCAFEBABE"))).isNotEmpty();
+        replicasPresentForKeyspace(keyspace).becomesTrue();
```

Comment:
Based on your comment below, I think I will need to separate out the validation between the true/false cases anyways.  Were you more against the naming strategy?  Or the gist of how the validation is done? (or both :P?)